tests/data/asa.json

```diff
@@ -3131,126 +3131,6 @@
     "sensors": {
         "discovery": {
             "sensors": [
-                {
```

Comment:
We see multiple sensors deleted. It probably shows that something is going wrong with the new code  preventing existing sensors to survive the change. You should end up with no sensor deleted if the new code is safe.